userguide/payment/includes/payment-overview.adoc

```diff
@@ -1,108 +1,40 @@
-That initial operation will create a https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/payment/api/Payment.java[Payment] and an initial https://github.com/killbill/killbill-api/blob/c243233be112165bf04a89a715b79112c6c5d1f7/src/main/java/org/killbill/billing/payment/api/PaymentTransaction.java[PaymentTransaction].
-The user can then submit additional requests for the same payment (capture, refund, ...) and each of those calls will result in additional `PaymentTransaction` attached to the same payment.
-The payments have a state which will drive the set of possible subsequent operations available. For instance, it is possible to make a capture call against a payment in a state `AUTH_SUCCESS` but it is not possible to make such an operation against a payment in an `AUTH_ERRORED` state.
-The set of payment state transitions are configured in an https://github.com/killbill/killbill/blob/master/payment/src/main/resources/org/killbill/billing/payment/PaymentStates.xml[xml file] and the Kill Bill payment subsystem is in charge to enforce the transitions. The following diagram below shows all the possible transitions:
-
-graphviz::assets/payment_states.dot[format=svg,align=center]
-
-In addition to the payment state transitions, each `PaymentTransaction` has a status to reflect the result of the operation. The `PaymentTransaction` status depends on the plugin result and it can be summarized below:
-
-|===
-|plugin result | payment transaction status | payment state | http response | description
-
-|PROCESSED
-|SUCCESS
-|{AUTH,CAPTURE,..}_SUCCESS
-| 201 - Created
-|The payment transaction went through and was successful
-
-|PENDING
-|PENDING
-|{AUTH,CAPTURE,..}_PENDING
-| 201 - Created
-|Successful asynchronous operation (e.g. ACH transfer) or multi-step call (e.g. 3D-Secure authorization)
+*Payment* - A Kill Bill record that encapsulates charging an end-user against a payment method. It also has a state that indicates whether the underlying transaction was successful or not.
 
-|ERROR
-|PAYMENT_FAILURE
-|{AUTH,CAPTURE,..}_FAILED
-| 402 - Payment Required
-|The payment transaction went through but failed (e.g insufficient funds)
 
-|CANCELED
-|PLUGIN_FAILURE
-|{AUTH,CAPTURE,..}_ERRORED
-| 502 - Bad Gateway
-|The payment transaction did not happen (e.g unable to connect to the provider, error in plugin, etc)
+=== How Payment Works
 
-|UNDEFINED, timeout or any exception
-|UNKNOWN
-|{AUTH,CAPTURE,..}_ERRORED
-| 503 - Service Unavailable
-504 - Gateway Timeout
-|The payment transaction may or not have succeeded, manual review needed
+The following diagram provides a high level overview of the payment system in Kill Bill:
 
-|===
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/payment-userguide/payment-system-overview.png[align=center]
```

Comment:
Kaui only talks to Kill Bill core via API. Maybe the arrow from Kaui should go to Kill Bill instead, otherwise it's a bit confusing?